pkgs/applications/misc/warpd/default.nix

```diff
@@ -26,21 +26,21 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ git ];
 
-  buildInputs =  [
+  buildInputs =  if !waylandSupport then [
```

Comment:
You should write this in the form `if FLAG then EXP1 else EXP2`, and not `if !FLAG then EXP2 else EXP1`.

Reply:
I hesitated to do it that way. Will do.